.github/ISSUE_TEMPLATE/4.docs_request.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Docs Request for an Update or Improvement'
+description: A task to update or improve Next.js documentation
+title: 'Docs: '
+labels:
+  - 'area: documentation'
```

Comment:
I think you'll probably want to triage these issues first before assigning them to `area:`, what we do for bug/feature request is use a `template` label so that we know which have not been triaged yet.

Reply:
Thats interesting. So a `template: docs` could mean that an issue has been created, and needs to be triaged before adding the `area: documentation` label and then the docs backlog. I like it. Will create a new label and change this